operator/src/main/resources/Operator.properties

```diff
@@ -208,6 +208,10 @@ exposing it.
 WLSDO-0014=The domain {0} introspect job failed, it will automatically retry in {1} seconds. Current number of  \
   retries = {2}
 WLSDO-0015="Retrying failed domain {0} introspect job, retry {1} of a maximum {2}."
+WLSDO-0016=DomainUID ''{0}'' exceeds maximum allowed length.
+WLSDO-0017=DomainUID ''{0}'' and cluster name ''{1}'' combination exceeds maximum allowed length.
```

Comment:
For each of these, it might be helpful for the administrator to also include the maximum allowed length as well as the 'combination' that is exceeding the length.  

For example:
```
The DomainUID ''{0}'' and cluster name ''{1}'' combination ''{2}'' exceeds maximum allowed length ''{3}".
```

Reply:
LGTM now